### PR TITLE
repository updates

### PR DESCRIPTION
**Pull request by laurenm-bbd**

### CODE REVIEW

**Comment by laurenm-bbd** [comment]
@OmerElgoni with the repo updates I have updated the UserController to use the repo pattern but can't commit into your branch until these changes have been merged. So just let me know before you do more work on the UserController, don't want to waster your time :)

File: totally-legit-horoscopes-api/Controllers/UsersController.cs [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+                    return starSign;
+                }
+            }
+
+            throw new InvalidOperationException("Date is not contained in any star sign");
+        }
+
+        private int calculateLifeNumber(DateTime dateOfBirth)
+        {
+            int year = dateOfBirth.Month;
```
Comment (thabovr) [inline totally-legit-horoscopes-api/Controllers/UsersController.cs]: Is this not dateOfBirth.year

File: totally-legit-horoscopes-api/Controllers/UsersController.cs [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+            int month = dateOfBirth.Month;
+            int day = dateOfBirth.Month;
```
Comment (thabovr) [inline totally-legit-horoscopes-api/Controllers/UsersController.cs]: Is this not dateOfBirth.Day

**Review by thabovr — COMMENTED** [review]
Left 2 comments regarding the datetime method

File: totally-legit-horoscopes-api/Controllers/UsersController.cs (hunk 2 above)
Comment (laurenm-bbd) [reply totally-legit-horoscopes-api/Controllers/UsersController.cs]: Ditto above comment :)

**Review by thabovr — APPROVED** [review]

**Pull request merged**